app/src/main/java/com/techcourse/dao/UserDao.java

```diff
@@ -1,121 +1,59 @@
 package com.techcourse.dao;
 
 import com.techcourse.domain.User;
+import java.util.List;
+import javax.sql.DataSource;
 import nextstep.jdbc.JdbcTemplate;
+import nextstep.jdbc.RowMapper;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import javax.sql.DataSource;
-import java.sql.Connection;
-import java.sql.PreparedStatement;
-import java.sql.ResultSet;
-import java.sql.SQLException;
-import java.util.List;
-
 public class UserDao {
 
     private static final Logger log = LoggerFactory.getLogger(UserDao.class);
 
-    private final DataSource dataSource;
+    private static final RowMapper<User> ROW_MAPPER = rs -> new User(rs.getLong(1),
+            rs.getString(2),
+            rs.getString(3),
+            rs.getString(4));
```

Comment:
```suggestion
    private static final RowMapper<User> ROW_MAPPER = rs -> new User(rs.getLong("id"),
            rs.getString("account"),
            // ...
```

숫자보다는, 실제 컬럼 값을 넣으면 가독성도 좋아지고, 순서에 대한 오류도 줄일 수 있을 것 같아요!